apps/web/components/ListViewCard/ListViewCard.tsx

```diff
@@ -0,0 +1,122 @@
+import { Box, Button, Checkbox, LinkV2, Text } from '@island.is/island-ui/core'
+import React from 'react'
+import * as styles from './ListViewCard.css'
+import { CTAProps } from '../ActionCategoryCard/ActionCategoryCard'
+
+type InfoItems = {
+  icon?: React.ReactNode
+  title: string
+}
+
+type CardProps = {
+  infoItems: Array<InfoItems>
+  onCheck?: (e: any) => void
```

Comment:
I don't see where this is used, however: do we not know the type for this property?

Reply:
This is used when the checkmark is checked, I removed the "e" parameter however